djstripe/models.py

```diff
@@ -541,20 +542,94 @@ def _attach_objects_hook(self, cls, data):
         if subscription:
             self.subscription = subscription
 
+        self._attach_invoice_items(cls, data)
+
+    def _attach_invoice_items(self, cls, data):
+        if not self.pk:
+            # InvoiceItems need a saved invoice because they're associated via
+            # a RelatedManager.  It might be better to make this pattern more
+            # generic with some sort-of _attach_objects_post_save_hook().
+            self.save()
+
+        cls._stripe_object_to_invoice_items(InvoiceItem, data, self)
+
+    @property
+    def plan(self):
+        """ Gets the associated plan for this invoice.
+
+        In order to provide a consistent view of invoices, it is necessary to
+        examine the invoice items for the associated subscription plan rather
+        than the top-level link.  The reason for this is that the subscription
```

Comment:
I'm not sure what this second sentence is meant to say.

Also, on a pedantic note, please use one space after a period.